Add automated testing and fix bug in question date.

aditya102 opened the pull request:
### Add automated testing for views and Published Question
1. Check for no Questions.
    * If no questions exits there should be an appropriate message.

2. Check for question with past date.
    * Question with past date should be displayed on the index page.

3. Check for question with future date.
     * Question with future date should not be on the index page.

4. Fixed Bug which shows the question with future date in current day.

aditya102 committed:
Add Auto testcase

waranlogesh commented on polls/models.py:
```diff
@@ -12,7 +12,8 @@ def __str__(self):
     def was_published_recently(self):
-        return self.pub_date >= timezone.now() - datetime.timedelta(days=1)
+        now = timezone.now()
+        return now - datetime.timedelta(days=1) <=self.pub_date <=now
```
```suggestion
        return now - datetime.timedelta(days=1) <= self.pub_date <= now
```
Please use flake8 proceeding forward to check your code. Google on what flake8 is and how it should be used.

waranlogesh commented on polls/tests.py:
```diff
@@ -1,2 +1,54 @@
+
+from .models import Question, Choice
+
+def create_question(question_text, days):
```
```suggestion

def create_question(question_text, days):
```
Two line breaks to separate outer class/method.

aditya102 replied:
Made the change

waranlogesh commented on polls/tests.py:
```diff
@@ -1,2 +1,54 @@
+        )
+
+    def test_future_question(self):
+    
```
```suggestion
```

waranlogesh commented on polls/tests.py:
```diff
@@ -1,2 +1,54 @@
+        self.assertQuerysetEqual(response.context['latest_question_list'], [])
+
+    def test_future_question_and_past_question(self):
+       
```
```suggestion
```

waranlogesh commented on polls/tests.py:
```diff
@@ -1,2 +1,54 @@
+
+    def test_two_past_questions(self):
 
+        create_question(question_text="Past question 1.", days=-30)
```
Remove the above line break.

waranlogesh commented on polls/tests.py:
```diff
@@ -1,2 +1,54 @@
+        create_question(question_text="Past question.", days=-30)
+        create_question(question_text="Future question.", days=30)
+        response = self.client.get(reverse('polls:index'))
+        self.assertQuerysetEqual(
```
Line break to separate asserts in all test cases.

waranlogesh commented:
PR title should be Add unit tests and fix a bug in question date

aditya102 committed:
Update code with flake8 guideline

waranlogesh commented on polls/tests.py:
```diff
@@ -1,2 +1,55 @@
+        )
+
+    def test_future_question(self):
+    
```
Remove this line break.

aditya102 committed:
Add unit tests and fix a bug in question date

aditya102 committed:
Add a line above assertQuery and Update Admin page

- Updated Name of Admin Page
- Added New Search feature And organize questions
- Added choice field in default
